### PR TITLE
Movie pages

### PR DESCRIPTION
**Pull request by whoisasey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "version": "0.1.0",
   "license": "0BSD",
   "scripts": {
-    "build": "gatsby build",
-    "develop": "gatsby develop",
+    "build": "GATSBY_EXPERIMENTAL_ROUTING_APIS=1 gatsby build",
```
Comment (sonalranjit) [inline package.json]: is this necessary?

File: gatsby-config.js [hunk 2]
```diff
@@ -31,5 +34,11 @@ module.exports = {
         url: process.env.HASURA_GRAPHQL_URL,
       }
     },
+    // {
+    //   resolve: 'gatsby-plugin-page-creator',
```
Comment (sonalranjit) [inline gatsby-config.js]: Could we remove these unused comments

File: src/components/helpers/Filter.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="films_filter">
+      <label htmlFor="filter">Filter By Year</label>
+			<select name="filter" id="filter"
+				// onChange={filter}
+			>
+				{years.map((el, index) => {
+					return (
+						<option value={el} key={index}>{ el}</option>
+					)
+				})}
```
Comment (sonalranjit) [inline src/components/helpers/Filter.js]: Is there way to format this better

File: src/components/helpers/allfilms.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+exports.allfilms = async () => {
+	  const data = await fetch("https://sonal-hasura.herokuapp.com/v1/graphql", {
+            "method": "POST",
+            "headers": {
+            "Content-Type": "application/json"
+            },
+                "body": "{\"query\":\"{\\n tnff_films {\\n                id\\n                title\\n                tnff_year\\n                location\\n                category\\n                release_date\\n                duration_mins\\n                language\\n                director\\n                producer\\n                synopsis\\n                director_bio\\n                film_img_url\\n                director_img_url\\n                other_info\\n            }\\n}\"}"
```
Comment (sonalranjit) [inline src/components/helpers/allfilms.js]: also this, is there a way to make this a prettier formatting, it's hard to read the query.

File: src/pages/films.js [hunk 5]
```diff
@@ -1,54 +1,113 @@
...
+
+  const getData = useCallback(async function () {
+    try {
+      const data = await fetch("https://sonal-hasura.herokuapp.com/v1/graphql", {
+        "method": "POST",
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": "{\"query\":\"{\\n tnff_films (order_by: {tnff_year: desc}) {\\n                id\\n                title\\n                tnff_year\\n                location\\n                category\\n                release_date\\n                duration_mins\\n                language\\n                director\\n                producer\\n                synopsis\\n                director_bio\\n                film_img_url\\n                director_img_url\\n                other_info\\n            }\\n}\"}"
+      });
```
Comment (sonalranjit) [inline src/pages/films.js:21]: same here, the formatting

File: src/pages/films.js [hunk 6]
```diff
@@ -1,54 +1,113 @@
...
+        // if (film.tnff_year === filmFilter) {
+      //     console.log(filmFilter)
+      //   console.log(film);
+      //   // filmsToRender.push(film)
+      //   }
+//       });
+//     }
+//   })
+// console.log(filmsToRender)
+// console.log(data)
```
Comment (sonalranjit) [inline src/pages/films.js]: unused code/comment could be removed

File: src/pages/films.js [hunk 7]
```diff
@@ -1,54 +1,113 @@
...
+return el.tnff_year
+})
+
+const years = [...new Set(yearsList)]
 
+return (
+<div className="wrapper">
+<Layout>
+  
+    {/* filter film by year */}
```
Comment (sonalranjit) [inline src/pages/films.js:72]: unnecessary brackets/comments ?

File: src/templates/FilmPage.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {BiWorld} from 'react-icons/bi'
+import {BsCalendar, BsPersonFill }from 'react-icons/bs'
+import {AiOutlineClockCircle} from 'react-icons/ai'
+import { FaLanguage } from 'react-icons/fa'
+import {RiMovieLine} from 'react-icons/ri'
+
+
+const FilmPage = (props) => {
+  const {	title,	synopsis,director_bio,	director,	duration_mins, language,release_date,	location,  director_img_url, 	other_info,	film_img_url,	tnff_year,	category} = props.pageContext.content
```
Comment (sonalranjit) [inline src/templates/FilmPage.js]: could be formatted better

File: src/components/helpers/allfilms.js (hunk 4 above)
Comment (whoisasey) [reply src/components/helpers/allfilms.js]: cannot separate these onto different lines

File: src/pages/films.js (hunk 5 above)
Comment (whoisasey) [reply src/pages/films.js:21]: removed allfilms.js, was a failed helper file

File: src/components/helpers/Filter.js (hunk 3 above)
Comment (whoisasey) [reply src/components/helpers/Filter.js]: looks fine on editor

**Review by sonalranjit — APPROVED** [review]
looks great

**Pull request merged**